### DISCUSSION TITLE
Full text search first indexing failed with "NoNodeAvailableException"

### DISCUSSION THREAD

**Discussion by vijdz** [nextcloud/all-in-one#3373] [Questions] [answered]
The installation is working like a charm with fulltextsearch disabled.

### Steps to reproduce

1. Enable FTS for the first time
2. Launch containers
=> Failure after 40K files and ~ 137 Mb usage 

### Expected behavior
Indexing should end without errors, there is enough RAM available.

### Actual behavior
Errors will continue until all users have been processed. The first index can never end completely.

### Host OS
Debian 12 with latest docker, lots of RAM and disk space

#### Nextcloud AIO version
27.1.1 and 27.2.1

#### Current channel
Stable

**Comment by szaimen** [comment, marked as answer]
Will be resolved in https://github.com/nextcloud/all-in-one/pull/3355

**Reply by vijdz** [reply]
I closed this, this was clearly not the right diff or title, I need to open an issue regarding the groupfolders thing, and may be it would be great to add the fix for #3354 in the docs.  
But I guess that PR was useless.  
Also I found another ES docs where they explicitly say Xms and Xmx must be the same
